src/lib.rs

```diff
@@ -46,3 +46,38 @@ pub mod features2d;
 
 #[cfg(feature = "gpu")]
 pub mod cuda;
+
+extern "C" {
+    fn c_drop(value: *mut c_void);
+}
+
+/// Structure that contains global cv functions
+pub struct Cv {}
```

Comment:
Can you clarify what `Cv` here is useful for?

Reply:
It must be removed